timetomodel/tests/test_forecasting.py

```diff
@@ -93,7 +93,9 @@ def test_rolling_forecast_with_refitting(caplog):
     end = DATA_START + timedelta(hours=190)
     forecasts, final_model_state = forecasting.make_rolling_forecasts(start, end, specs)
     expected_values = specs.outcome_var.load_series(
-        expected_frequency=timedelta(hours=1)
+        time_window=(start, end),
```

Comment:
Can you tell me why we need to add `time_window` and `check_time_window` to this test? Or are they only being added so that this test runs by the associated code?

Reply:
The latter.